### ISSUE TITLE
server crash when parsing request

### ISSUE THREAD

**Issue by vrkuvrku** [alosaur/alosaur#200]
One can easily (even accidentally) crash whole server process by a malformed request, e.g. using https:// instead of http://

```
curl https://localhost:8000

$ deno run -A examples/basic/app.ts 
Http: invalid HTTP method parsed
    at async HttpConn.nextRequest (ext:deno_http/01_http.js:101:21)
    at async handleFullServer (file:///Users/martin/lab/alosaur/src/server/handle-native-request.ts:73:17) {
  name: "Http"
}
$ 
```

The excepted behaviour should be just ignoring such requests, by catching exceptions when iterating thru `for await (const request of requests)`

```
index c890f82..61fae2d 100644
--- a/src/server/handle-native-request.ts
+++ b/src/server/handle-native-request.ts
@@ -66,7 +66,16 @@ async function handleFullServer<TState>(
   metadata: MetadataArgsStorage<TState>,
 ) {
   const requests = Deno.serveHttp(conn);
-  for await (const request of requests) {
+
+  while (true) {
+    let request;
+    try {
+      request = await requests.nextRequest();
+    } catch (error) {
+      console.error(error);
+      break;
+    }
+
     const respondWith = respondWithWrapper(request.respondWith, conn);
 
     metadata.container.register(SERVER_REQUEST, { useValue: request });
```

**Comment by irustm** [comment]
Solved in v1.0.0

**Comment by vrkuvrku** [comment]
Perfect, I just verified it, thank you!
